glass/src/libnt/native/cpp/NTField2D.cpp

```diff
@@ -126,7 +126,7 @@ void NTField2DModel::Update() {
   for (auto&& event : m_poller.ReadQueue()) {
     if (auto info = event.GetTopicInfo()) {
       // handle publish/unpublish
-      auto name = wpi::drop_front(info->name, m_path.size());
+      auto name = wpi::remove_prefix(info->name, m_path).value();
```

Comment:
```suggestion
      auto name = wpi::remove_prefix(info->name, m_path).value_or("");
```
This will cause it to just continue rather than crashing with an exception.